templates/subject.html

```diff
@@ -95,6 +95,17 @@
 
     </tbody>
   </table>
+
+  {% for page_num in articles.iter_pages() %}
+    {% if page_num %}
+      {% if articles.page == page_num%}
+        <a class="btn btn-info mb-4" href="{{ url_for('get_subject', subjectID=subject_id, page=page_num)}}">{{ page_num }}</a>
+      {% else %}
+        <a class="btn btn-outline-info mb-4" href="{{ url_for('get_subject', subjectID=subject_id, page=page_num)}}">{{ page_num }}</a>
+      {% endif %}
+    {% endif %}
+{% endfor %}
+
```

Comment:
Maybe a quick overview comment to explain what is the purpose of this block.

Thanks.

Reply:
@hema-001 
This block is for displaying those page numbers below the table. Here we are iterating over the pages using that `iter_pages()` method. Those page numbers are links to the pages that display the items on that page and the statement `{% if articles.page == page_num%}` is to check if the page we are in the iteration is the current page. Therefore, the button for the active page is filled in, to distinguish it from the other buttons.

![buttons](https://user-images.githubusercontent.com/45811924/79832706-98556b00-83dc-11ea-8e24-c498955dce1b.JPG)